pip_review/__main__.py

```diff
@@ -76,20 +78,15 @@ def parse_args():
     parser.add_argument(
         '--auto', '-a', action='store_true', default=False,
         help='Automatically install every update found')
-    parser.add_argument(
-        '--editables', '-e', action='store_true', default=False,
-        help='Also include editable packages in PyPI lookup')
-    parser.add_argument(
-        '--user', '-u', action='store_true', default=False,
-        help='Only output packages installed in user-site.')
-    parser.add_argument(
-        '--local', '-l', action='store_true', default=False,
-        help='If in a virtualenv that has global access, do not output '
-             'globally-installed packages')
-    parser.add_argument(
-        '--pre', '-p', action='store_true', default=False,
-        help='Include pre-release and development versions')
-    return parser.parse_args()
+
+
+    parsed, unknown = parser.parse_known_args() #this is an 'internal' method
```

Comment:
No, this is *not* an internal method, even if you quote it. It is a public, well-documented, official part of the standard library which was included precisely for our purpose. I'm glad you found it.

`unknown` is a rather confusing name for a variable. I did some renaming and removed your comment in db0f0fca.